### ISSUE TITLE
Pagination does not work

### ISSUE THREAD

**Issue by mayorgatellez** [echevemaster/fudcon#8]
The speakers and sessions pages don't show the entire lists. It looks like there should be buttons to navigate through the pages. If you scroll down you can't see all the items.

**Comment by echevemaster** [comment]
Yes it's a bug. I'm finishing other features, so I hope fix it in two days and merge.
